### PR TITLE
fix: Rebuild get.sh with testing

### PR DESCRIPTION
**Pull request by mawelo**
Changes to be committed:
	modified:   installer/get.sh

re-write the installer script and tested on my QC
with CorOS:  2.2.2

### CODE REVIEW

File: installer/get.sh [hunk 1]
```diff
@@ -1,12 +1,39 @@
 #!/bin/sh
+
+# 2024/06/19/ml tested with CorOS:  2.2.2
+
+readonly OPPATH='/media/p4/OpenCortex'
```
Comment (JudahZF) [inline installer/get.sh:3]: Suggestion: Can we use OCPATH instead?

File: installer/get.sh [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+readonly BINDNAME="${OPPATH}/bin"
+readonly BREWDNAME="${OPPATH}/ZenBrew"
+readonly URI="https://github.com/OpenCortex-Research/ZenBrew/releases/download/V1.0.0"
+readonly curlbin="/bin/wget"
+
+test -x ${curlbin} || {
+	echo "ERROR: unable t oexecute ${curlbin}"
```
Comment (JudahZF) [inline installer/get.sh]: typo: to execute

File: installer/get.sh [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+${curlbin} ${URI}/zenbrew && \
+strings ./zenbrew|grep 'GOARM=' 1>/dev/null 2>&1 || {
+	echo "ERROR: failed to download the right binary 'zenbrew' for ARM architecture"
+	exit 100
+} && \
+chmod +x ${BINDNAME}/zenbrew && \
+cd ${BREWDNAME} && \
+${curlbin} ${URI}/settings.json 2>/dev/null && \
+test -z settings.json && {
+	echo "ERROR: fiel size of settings.json is zero"
```
Comment (JudahZF) [inline installer/get.sh]: typo: file

**Comment by mawelo** [comment]
need to fix some typos

File: installer/get.sh [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
-curl -O https://github.com/OpenCortex-Research/ZenBrew/releases/download/V1.0.0/settings.json
-cd /media/p4/OpenCortex/bin
-./zenbrew install zenbrew
+mkdir -p ${OPPATH} && \
+mkdir -p ${BINDNAME} && \
+mkdir -p ${BREWDNAME} && \
+cd ${BINDNAME} && \
+rm -f zenbrew && \
+${curlbin} ${URI}/zenbrew && \
+strings ./zenbrew|grep 'GOARM=' 1>/dev/null 2>&1 || {
```
Comment (JudahZF) [inline installer/get.sh:21]: nitpick: it's a bit more readable with spaces around the pipe. zenbrew | grep

**Review by JudahZF — CHANGES_REQUESTED** [review]
Could you just fix a few typos and some formatting, then I'll merge it.

**Review by JudahZF — APPROVED** [review]
Looks good!

**Pull request merged**